core/types/gen_tx_json.go

```diff
@@ -7,8 +7,8 @@ import (
 	"errors"
 	"math/big"
 
-	"github.com/ethereum/go-ethereum/common"
-	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/harmony-one/harmony/vendor/github.com/ethereum/go-ethereum/common"
```

Comment:
can you please confirm that we have to specify the code from vendor? @chaosma is the right import @astralblue ?

Reply:
fixed